ra-optimisation/linear-rao/src/main/java/com/farao_community/farao/linear_rao/config/LinearRaoConfigurationUtil.java

```diff
@@ -1,89 +0,0 @@
-/*
- * Copyright (c) 2020, RTE (http://www.rte-france.com)
- * This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/.
- */
-package com.farao_community.farao.linear_rao.config;
-
-import com.farao_community.farao.commons.FaraoException;
-import com.farao_community.farao.data.crac_api.Crac;
-import com.farao_community.farao.data.crac_loopflow_extension.CracLoopFlowExtension;
-import com.farao_community.farao.rao_api.RaoParameters;
-
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Objects;
-
-/**
- * Utility class designed to check availability of all plugins listed in the
- * RAO parameters, and to interpret them in different contexts of the linear
- * RAO.
- *
- * @author Philippe Edwards {@literal <philippe.edwards at rte-france.com>}
- */
-public final class LinearRaoConfigurationUtil {
-
-    private LinearRaoConfigurationUtil() {
-        throw new AssertionError("Utility class should not be instanciated");
-    }
-
-    /**
-     * Validates RAO parameters compatibility with platform available plugins.
-     * Return a list of errors, that is empty if the configuration is correct.
-     *
-     * @param raoParameters RAO parameters
-     * @return a list of configuration issues
-     */
-    public static List<String> checkLinearRaoConfiguration(RaoParameters raoParameters, Crac crac) {
-        List<String> errors = new ArrayList<>();
-
-        // Check that correct extension is provided
-        // Return directly if the extension is not provided
-        try {
-            getLinearRaoParameters(raoParameters);
-        } catch (FaraoException e) {
-            errors.add(e.getMessage());
-            return errors;
-        }
-
-        // loopflows extension vs. loopflow parameters
-        if (raoParameters.isRaoWithLoopFlowLimitation() && Objects.isNull(crac.getExtension(CracLoopFlowExtension.class))) {
```

Comment:
loopflow parameter consistency checks are disappeared after the refactoring?

Reply:
There is a check in RaoUtil.intiRaoData() and this method is called at the beginning of RAOs so there is a data consistency issue, it should be detected here

Reply:
ok. removed check for virtual cost and adjustment coeff. might be a minor problem if virtual cost is negative. but ok